editor/src/clj/editor/model.clj

```diff
@@ -443,6 +445,8 @@
           (g/fnk [material-scene-infos]
             (model-scene/make-material-name->material-scene-info material-scene-infos)))
 
+  (property create-go-bones g/Bool (default false))
```

Comment:
We use defaults from protobuf (and the default there is true). Is this intentional? Looks like a breaking change...

Reply:
Also, for defaults, we don't typically hard-code them, but instead look them up, e.g. `(protobuf/default ModelProto$ModelDesc :create-go-bones)`